src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -941,7 +941,9 @@ static void glissando(const Glissando* gli, int number, bool start, Notations& n
         break;
     }
     tagName += QString(" number=\"%1\" type=\"%2\"").arg(number).arg(start ? "start" : "stop");
-    tagName += color2xml(gli);
+    if (gli->lineColor() != engravingConfiguration()->defaultColor()) {
+        tagName += QString(" color=\"%1\"").arg(QString::fromStdString(gli->lineColor().toString()).toUpper());
```

Comment:
I believe this could be done in color2xml(), check my backport to 3.x

Reply:
indeed the more elegant solution, thanks.

Reply:
Pretty close to your earlier attempt